### PR TITLE
[DOC] Correct argument for optimizer `ranger` in `Temporal Fusion Transformer` tutorial

### PR DESCRIPTION
**Pull request by fnhirwa**
fixes error in the tutorial  which was causing examples to fail.
fixes https://github.com/sktime/pytorch-forecasting/issues/1722

### CODE REVIEW

**Review by fkiraly — CHANGES_REQUESTED** [review]
the notebook CI failed, looks like a timeout? I will restart

**Comment by fnhirwa** [comment]
The issue was the training of the `TFT` model which is taking longer than `nbsphinx` timeout time which is `600ms` I run the example locally and disabled notebooks run on CI in `conf.py` You can check the notebook output from ReviewNB before merging.

File: docs/source/conf.py [hunk 1]
```diff
@@ -180,3 +180,8 @@ def setup(app: Sphinx):
 suppress_warnings = [
     "autosummary.import_cycle",
 ]
+
+# -----------nbsphinx extension ----------
+nbsphinx_execute = "never"  # always
```
Comment (yarnabrina) [inline docs/source/conf.py:190]: If you never run the notebooks, do the below the configurations have any effect?

File: docs/source/tutorials/stallion.ipynb
Comment (yarnabrina) [inline docs/source/tutorials/stallion.ipynb:1]: I'm unable to see this diff in Github (it saus unable to render rich diff), and ReviewNB also seems to lack the notebook.

File: docs/source/conf.py (hunk 1 above)
Comment (fnhirwa) [reply docs/source/conf.py:190]: I found that there is a workflow run for testing the notebook and it was one causing the timeout error, this is for building the docs.

File: docs/source/tutorials/stallion.ipynb
Comment (fnhirwa) [reply docs/source/tutorials/stallion.ipynb:1]: I'm not quite sure about that, I logged in with my github and was able to see diff on ReviewNB
image

File: docs/source/conf.py (hunk 1 above)
Comment (yarnabrina) [reply docs/source/conf.py:190]: If I am not wrong, `nbsphinx_execute = "never"` will ensure that RTD will not execute the example notebooks while building the docs (ref. https://nbsphinx.readthedocs.io/en/0.9.5/never-execute.html). If that is the case, whether that execution need to ignore errors or not (`nbsphinx_allow_errors = False`) or how long execution takes (`nbsphinx_timeout = 600`) should not matter.

Does that make sense? Let me know if I am missing something, or misunedrstanding the purpose of these configurations.

Comment (fnhirwa) [reply docs/source/conf.py:190]: You're right, these extra configurations are meaningless.
should we keep the `nbsphinx_execute = "never"`? as we run notebooks in this workflow to test if they don't fail.https://github.com/sktime/pytorch-forecasting/blob/e08f4e6d23737c36949d704215ccbbbaa84fc1b6/.github/workflows/test.yml#L17

File: docs/source/tutorials/stallion.ipynb
Comment (yarnabrina) [reply docs/source/tutorials/stallion.ipynb:1]: Not sure why it's not working for me, but not a big deal. I checked it from branch it directly.

I am not what change of yours is making the predictions worse. In `main` branch, the predicted orange line and shaded area seemed to contain the blue actuals pretty well, but in the modified one the gaps seem to have increased notably. Do you agree with observation? Any guess what's causing it (seed remained same at 42 in both cases)?

File: docs/source/conf.py (hunk 1 above)
Comment (yarnabrina) [reply docs/source/conf.py:190]: I believe `never` is fine, these executions increase RTD time significantly. Let's skip in `sktime` too for that matter, if @fkiraly agree as well (I have seen too many timeouts recently).

File: docs/source/tutorials/stallion.ipynb
Comment (fnhirwa) [reply docs/source/tutorials/stallion.ipynb:1]: I'm not sure about the cause as I can see even the learning_rate got changed from `0.041` to `0.009`.  I think I'll need to check the logs and see where it is going wrong

Comment (fnhirwa) [reply docs/source/tutorials/stallion.ipynb:1]: I rerun the notebook now the losses seems to have been reduced

**Review by fkiraly — APPROVED** [review]
Great!

Question, non-blocking: can we perhaps change the examples that they are still meaningful, but have lower runtime?

**Comment by fnhirwa** [comment]
> Great!
> 
> Question, non-blocking: can we perhaps change the examples that they are still meaningful, but have lower runtime?

Yeah, this sounds great we can do it.

**Comment by fnhirwa** [comment]
I think the best approach is to have a broader task for reworking the examples as the ones available now seem to be written a long time ago. With this we can ensure that we have low runtime working examples.
